client/src/actions/postActions.js

```diff
@@ -50,7 +50,7 @@ export const getSinglePost = (postId, history) => dispatch => {
 // Post Comment
 export const sendComment = comment => dispatch => {
   axios
-    .post("/api/post/comment/${postid}", comment)
+    .post(`/api/post/comment/${postid}`, comment)
```

Comment:
postId is  missing. It is not passed as parameter.

Reply:
I updated it, I hadn't pushed the changes I made in postActions.js